gatsby-config.js

```diff
@@ -26,6 +26,14 @@ module.exports = {
           include: /\.inline\.svg$/
         }
       }
+    },
+    {
+      resolve: 'gatsby-source-ghost',
+      options: {
+        apiUrl: 'https://blog.hermez.io',
+        contentApiKey: 'a0bbfb0931ce4534811bee1acf',
```

Comment:
This might need to be an env variable

Reply:
Yep! This needs to be an env var. Make sure you rewrite the git history when doing it @mmilenkovic, so the `contentApiKey` can't be seen at any old commit.

Reply:
I will delete this branch and do commit in a new one, so we don't need to worry about history.

Reply:
contentApiKey regenerated. Secret updated. Security restored. 🥳